pkgs/development/tools/language-servers/jdt-language-server/update.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash --pure -p curl cacert libxml2 yq nix jq
+
+set -o errexit
+set -o nounset
```

Comment:
Usually just written `set -euo pipefail` (which also enables `pipefail` :-) ).

```suggestion
set -euo pipefail
```